code/ShareCare.php

```diff
@@ -197,8 +197,8 @@ public function EmailShareLink()
             return false;
         }
         $pageURL = $this->owner->AbsoluteLink();
-        $subject = rawurlencode('Thought you might like this');
-        $body = rawurlencode("Thought of you when I found this: $pageURL");
+        $subject = rawurlencode(_t('ShareCare.EmailSubject'));
+        $body = rawurlencode(_t('ShareCare.Body') . " " . $pageURL);
```

Comment:
Can you please revise this to pass the $pageURL variable in to the translation function. See third example here: https://docs.silverstripe.org/en/3/developer_guides/i18n/#usage-in-php-files